dev-environment/nussknacker/opt/prinz-sample/prinz-application.conf

```diff
@@ -0,0 +1,37 @@
+# This config is a modified copy of /nussknacker-dist/src/universal/conf/docker-application.conf
+base: { include "base-application.conf" }
+
+environment: "demo"
+
+categoriesConfig {
+  "FraudDetection": "streaming",
+  "Recommendations": "streaming"
+}
+
+processTypes {
+  "streaming": {
+    engineConfig: ${base.flinkEngineConfig}
+    modelConfig: {
+      classPath: [
+        "/opt/prinz/prinz_2.12-0.0.1-SNAPSHOT.jar",
+        "/opt/prinz-sample/prinz-sample_2.12-0.0.1-SNAPSHOT.jar"
+      ]
+    }
+  }
+}
+
+metricsSettings: {
+  url: ${base.grafanaUrl}"/dashboard/db/$dashboard?theme=dark&var-processName=$process&var-env="${environment}
+  defaultDashboard: "flink-esp"
+  processingTypeToDashboard: {
+    "streaming": "flink-esp"
+  }
+}
+
+# TODO: lightbend config can't include files on root level - move nussknacker config on nk level and get rid of this below
```

Comment:
TODO shouldn't be merged - open an isssue or solve in this commit if this is yours,  remove if unuesed

Reply:
This TODO comes from Nussknacker. Let's keep it here for reference and update configs when they do the fix.